src/main/kotlin/no/skatteetaten/aurora/gobo/security/currentUser.kt

```diff
@@ -16,21 +16,25 @@ const val GUEST_USER_ID = "anonymous"
 const val GUEST_USER_NAME = "Gjestebruker"
 val ANONYMOUS_USER = User(GUEST_USER_ID, GUEST_USER_NAME)
 
-suspend fun DataFetchingEnvironment.checkValidUserToken() {
+suspend fun DataFetchingEnvironment.checkValidUserToken(): User {
     token()
+    val user = currentUser()
     if (currentUser() == ANONYMOUS_USER) {
```

Comment:
Ikke kalle currentUser to ganger?

Reply:
fikset https://github.com/Skatteetaten/gobo/pull/209/commits/22211bfd6c606d7c740aa917834fc735d56ce3d4